src/components/MainSwiper/MainSwiper.module.scss

```diff
@@ -0,0 +1,52 @@
+@import '@/styles/index';
+
+.swiper {
+  width: 100%;
+  height: min-content;
+  overflow: visible;
+
+  @include on-desktop {
+    overflow: hidden;
+  }
```

Comment:
Overflow must be hidden on all devices because if overflow is visible occupies a lot of space and breaks page  
![image](https://github.com/user-attachments/assets/182528ca-a1b4-487f-8ec5-be2c94780191)